test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -483,33 +492,56 @@ def verify_option_exists(self, option:str):
      
-    def verify_permission_screen_title(self, option:str):
+    def verify_welcome_screen_title(self, option:str):
         if option=="Permissions":
             title = get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_TITLE.value).text
             verify_text(option, str(title))
+        elif option=="Tokens":
+            title = get_obj(CommunityTokensComponents.TOKENS_SCREEN_TITLE.value).text
+            verify_text(option, str(title))
              
-    def verify_welcome_permission_image(self):
-        path = get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_IMAGE.value).source.path
-        verify_text_contains(str(path), "permissions2_3")
+    def verify_welcome_image(self, option:str):
+        if option=="Permissions":
+            path = get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_IMAGE.value).source.path
+            verify_text_contains(str(path), "permissions2_3")
+        elif option=="Tokens":
+            path = get_obj(CommunityTokensComponents.TOKENS_SCREEN_IMAGE.value).source.path
+            verify_text_contains(str(path), "mint2_1")
     
-    def verify_welcome_settings_title(self):
-        verify_equals(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_TITLE.value).text, "Permissions")    
+    def verify_welcome_title(self, option:str):
+        if option=="Permissions":
+            verify_equals(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_TITLE.value).text, "Permissions")    
+        elif option=="Tokens":
+            verify_equals(get_obj(CommunityTokensComponents.TOKENS_SCREEN_MAIN_TITLE.value).text, "Community tokens")    
     
-    def verify_welcome_settings_subtitle(self):
-        verify_equals(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_SUBTITLE.value).text, "You can manage your community by creating and issuing membership and access permissions")
+    def verify_welcome_subtitle(self, option:str):
+        if option=="Permissions":
+            verify_equals(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_SUBTITLE.value).text, "You can manage your community by creating and issuing membership and access permissions")
+        elif option=="Tokens":
+            verify_equals(get_obj(CommunityTokensComponents.TOKENS_SCREEN_SUBTITLE.value).text, "You can mint custom tokens and import tokens for your community")
+            
         
-    def verify_welcome_settings_checklist(self, list: list):
-        checklist = []
-        checklist.append(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT1.value).text)
-        checklist.append(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT2.value).text)
-        checklist.append(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT3.value).text)
+    def verify_welcome_settings_checklist(self, option:str, list: list):
```

Comment:
Here it is not needed the if/else statement since we are just comparing the given list:
```suggestion
    def verify_welcome_settings_checklist(self, option:str, list: list):
    checklist = []
        checklist.append(get_obj(CommunityWelcomePageComponents.WELCOME_SCREEN_CHECKLIST_ELEMENT1.value).text)
        checklist.append(get_obj(CommunityWelcomePageComponents.WELCOME_SCREEN_CHECKLIST_ELEMENT2.value).text)
        checklist.append(get_obj(CommunityWelcomePageComponents.WELCOME_SCREEN_CHECKLIST_ELEMENT3.value).text)
        
        # Check if the lists are of equal length
        if len(checklist) != len(list):
            return False
        
        # Check if the lists have the same elements in the same order
        for i in range(len(checklist)):
            if checklist[i] != list[i]:
                return False
```